tests/trainer/logging/test_logger_connector.py

```diff
@@ -425,3 +425,21 @@ def test_dataloader(self):
     )
     trainer.fit(model)
     trainer.test(model, ckpt_path=None)
+
+
+def test_logging_to_progress_bar_with_reserved_key(tmpdir):
```

Comment:
@tchaton @Borda Where should I put this test? I'm not familiar with how the logging tests are structured.

Reply:
tests/trainer/logging_process ?

Reply:
this folder does not exist on master, only dev branch

Reply:
oh, I see.... :]

Reply:
Maybe callback/progress ?